File: app/src/main/java/com/neome/java/api/meta/base/dto/DefnDtoLayoutCardItem.java
// neome.ai API - do not change
//

package com.neome.java.api.meta.base.dto;

import com.neome.java.api.meta.base.Types.EnumDefnThemeImageCorner;
import com.neome.java.api.meta.base.Types.EnumDefnThemeImageRenderingMode;
import com.neome.java.api.meta.base.Types.MetaIdField;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class DefnDtoLayoutCardItem
{
  @Nullable
  public DefnDtoLayoutCardItemLine fifthLine;

  @Nullable
  public DefnDtoLayoutCardItemLine firstLine;

  @Nullable
  public DefnDtoLayoutCardItemLine fourthLine;

  @Nullable
  public DefnDtoColor imageBackgroundColor;

  @Nullable
  public EnumDefnThemeImageCorner imageCornerVar;

  @Nullable
  public Long imageHeight;

  @Nullable
  public Long imageHeightVar;

  @Nullable
  public EnumDefnThemeImageRenderingMode imageRenderingMode;

  @Nullable
  public Long imageWidth;

  @Nullable
  public Long imageWidthVar;

  @Nullable
  public MetaIdField[] mediaFieldIdSet;

  @Nullable
  public DefnDtoMedia[] mediaVarSet;

  @Nullable
  public DefnDtoLayoutCardItemLine secondLine;

  @Nullable
  public DefnDtoLayoutCardItemLine thirdLine;
}

File: app/src/main/java/com/neome/api/meta/base/dto/DefnDtoLayoutCardItem.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.EnumDefnThemeImageCorner
import com.neome.api.meta.base.Types.EnumDefnThemeImageRenderingMode
import com.neome.api.meta.base.Types.MetaIdField

interface DefnDtoLayoutCardItem {
    val fifthLine: DefnDtoLayoutCardItemLine?
    val firstLine: DefnDtoLayoutCardItemLine?
    val fourthLine: DefnDtoLayoutCardItemLine?
    val imageBackgroundColor: DefnDtoColor?
    val imageCornerVar: EnumDefnThemeImageCorner?
    val imageHeight: Long?
    val imageHeightVar: Long?
    val imageRenderingMode: EnumDefnThemeImageRenderingMode?
    val imageWidth: Long?
    val imageWidthVar: Long?
    val mediaFieldIdSet: List<MetaIdField>?
    val mediaVarSet: List<DefnDtoMedia>?
    val secondLine: DefnDtoLayoutCardItemLine?
    val thirdLine: DefnDtoLayoutCardItemLine?
}
